####################
# DeepGuider Status
####################

# current pose in UTM (meter)
float64 x
float64 y

# current pose in LatLon (degree)
float64 lat
float64 lon

# pose confidence (0 ~ 1)
float64 confidence

# system status (on/off)
bool dg_shutdown

# timestamp (in seconds)
float64 timestamp
